.github/workflows/ci.yml

```diff
@@ -132,7 +135,7 @@ jobs:
       - uses: actions/download-artifact@v4
         with:
           name: macos
-          path: native/target/native/x86_64-darwin/bin
+          path: native/target/native/${{ runner.arch }}-darwin/bin
```

Comment:
Sadly here we'd need to leave it hardcoded -- `arm64-darwin` and `x86_64-darwin`

The idea is that we cross build in the build step and upload mac artifacts; and publish via linux artefacts but download mac cached dylibs and put them in a specific folder so its [packed up on publish into the natives jar](https://github.com/PDAL/java/blob/main/build.sbt#L89-L91)

Reply:
Yeah. I'm fumbling through it. I'll create a new PR once it's working.